btree/btree.c

```diff
@@ -7099,8 +7102,15 @@ static int64_t btree_del_kv_tick(struct m0_sm_op *smop)
 		}
 
 		lev = &oi->i_level[oi->i_used];
+
 		node_slot.s_node = lev->l_node;
 		node_slot.s_idx  = lev->l_idx;
+		node_slot.s_rec  = REC_INIT(&p_key, &ksize, &p_val, &vsize);
+
+		node_rec(&node_slot);
+		node_slot.s_rec.r_flags = M0_BSC_SUCCESS;
+		rc = bop->bo_cb.c_act(&bop->bo_cb, &node_slot.s_rec);
```

Comment:
The callback needs to be called only if (bop->bo_cb.c_act != NULL)

Reply:
done.